README.md

```diff
@@ -17,6 +17,16 @@ and navigate to `localhost:8080/pkg/github.com/in-toto/in-toto-golang/in_toto/`
 
 
 ## Not (yet) supported
+
+This golang implementation was focused on verification on admission controllers
+and kubectl plugins. As such, it focused on providing a strong, auditable set
+of core functions rather than a broad and (possibly) unstable feature set. In
+other words, we believe that the current feature set is stable enough for
+production use.
+
+If any of these features are necessary for your usecase please let us know and
```

Comment:
I think *usecase* is usally written *use case* or *use-case*.

Reply:
It's usually _use case_.